### PR TITLE
Tutorial for setting up a private server in a headless environment

### PR DESCRIPTION
**Pull request by tedivm**

### CODE REVIEW

**Comment by o4kapuk** [comment]
@tedivm hey, nice article!
Could you please fix line 156 (should be sudo su screeps, not just su screeps) and specify explicitly that we have to leave 'screeps' user session before doing the 'Setup Service' step?

**Comment by tedivm** [comment]
Changes made.

**Comment by tedivm** [comment]
Updated to use `npx`.

File: source/contributed/ps_ubuntu.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ During this step you will need a Steam API Key, which you can obtain [from here]
 mkdir ~/world
 cd ~/world
 npm install screeps
-./node_modules/screeps/bin/screeps.js init
+npx screeps init
```
Comment (PostCrafter) [inline source/contributed/ps_ubuntu.md]: You should make sure that `npx` is actually installed, so it might be a good idea to install it here as well.

**Comment by tedivm** [comment]
Anything else you'd like changed? We've pinned this in #servers on slack and have been pointing it out to people, so it's gotten quite a few people who have run through it themselves.

**Comment by artch** [comment]
Published in the [docs](http://docs.screeps.com/contributed/ps_ubuntu.html) and mentioned in the main project [README](https://github.com/screeps/screeps#storage). 5 Subscription Tokens are added to your account. Thanks for contributing!

**Pull request merged**